From 4cfe90c14463bece5c77214a262be55c8efe90e2 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Thu, 15 Mar 2012 15:46:02 +0800
Subject: [PATCH 1447/2463] ENGR00176154 mx6q sabresd: change the position
 angle of board and LVDS

The LVDS display direction should be aligned with camera sensor.
So we rotate it with 180 degree.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabresd.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabresd.c b/arch/arm/mach-mx6/board-mx6q_sabresd.c
index 2192abb..47e9211 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabresd.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabresd.c
@@ -198,8 +198,8 @@
 #define SABRESD_ELAN_INT	IMX_GPIO_NR(3, 28)
 
 static struct clk *sata_clk;
-static int mma8451_position = 3;
-static int mag3110_position;
+static int mma8451_position = 1;
+static int mag3110_position = 2;
 static int max11801_mode = 1;
 static int enable_lcd_ldb;
 
-- 
1.7.7.4

